src/buttons/button.scss

```diff
@@ -312,14 +312,34 @@
 }
 
 // Icon-only buttons
-.btn-icon {
-  display: inline-block;
-  // stylelint-disable-next-line primer/spacing
-  padding: 7px;
-  line-height: $lh-condensed-ultra;
-  vertical-align: middle;
+// default mobile sizing
+.btn--icon {
```

Comment:
Should we keep the class name as `.btn-icon` for now to stay consistent with the rest?

Or make it a completely new/separate component called `.IconButton`. Maybe depends a bit what PVC/PRC will name it?

At some point I assume we'll want to rename `.btn` to `.Button` to match PRC and our naming convention.

Reply:
@simurai I'd recommend to make a new/separate component called `IconButton` to match our new naming conventions. This would also make it be easier to introduce this new style without requiring a migration of all existing `btn-icon` instances at the same time.

:v:

> At some point I assume we'll want to rename .btn to .Button to match PRC and our naming convention.

👍

Reply:
I'm thinking about moving these styles to an entirely new file, using `.IconButton` (because yes, we should rewrite `btn` to be `Button` next) essentially treating this as a new component vs. refactor. Issue here is that `IconButton` still shares things with `btn`, mostly variants. This is why I much prefer to handle rewrites holistically from the top-

`Button` -- `IconButton` -- `AccessoryButton` (or whatever this ends up being named)

But we started with IconButton here. Maybe we can take a step back and think about `Button` as well.

Reply:
>  Issue here is that `IconButton` still shares things with `btn`, mostly variants.

Ouuu.. right! Using `<button class="btn IconButton">` does feel a bit weird.

> But we started with IconButton here. Maybe we can take a step back and think about `Button` as well.

I guess we could use `<button class="IconButton">` stand-alone by including all the `btn` styles. Downside is that it would increase file size compared to using a "variation" like `<button class="btn IconButton">`.

Hmm.. visually, `IconButton` still looks pretty much like a default `btn`, just with a fixed size meant for icons. Because of that we might still should keep it as a "variation"? 🤔  And in that case with all the renaming done, it would become `<button class="Button Button--icon">`. And then, yes, it would make sense to start with the default `Button`. 👍 

How urgent is having this "icon button"?